packages/core/src/Cardano/types/Script.ts

```diff
@@ -23,15 +31,20 @@ export enum NativeScriptType {
  * key hash.
  */
 export interface RequireSignatureScript {
+  /**
+   * Script type.
+   */
+  __type: ScriptType.Native;
+
   /**
    * The hash of a verification key.
    */
   keyHash: Ed25519KeyHash;
 
   /**
-   * Script type.
+   * The native script kind.
    */
-  __type: NativeScriptType.RequireSignature;
+  kind: NativeScriptType.RequireSignature;
```

Comment:
These changes are in the wrong commit too

Reply:
changes moved to appropriate commit

Reply:
It wasn't a clean separation, so I've just squashed them again with a more detailed commit message
05a83b9bba28b0d7fc958d5b30703ad3121749c2